onnxruntime/core/optimizer/layer_norm_fusion.cc

```diff
@@ -255,6 +254,15 @@ Status LayerNormFusion::ApplyImpl(Graph& graph, bool& modified, int graph_level,
                                           layer_norm_input_defs,
                                           {}, {}, kOnnxDomain);
 
+    // Get constant "epsilon" from "Add2" node if available. Else, default value will be used. 
+    const ONNX_NAMESPACE::TensorProto* tensor_proto = graph_utils::GetConstantInitializer(graph, add2_node.MutableInputDefs()[1]->Name());
+    if (tensor_proto != nullptr) {
+      if (tensor_proto->data_type() == ONNX_NAMESPACE::TensorProto_DataType_FLOAT) {
```

Comment:
Might need add FP16 data type support. You can use a FP16 model to test whether it is needed.